mass-stop-start-cf-apps/README.md

```diff
@@ -0,0 +1,62 @@
+# Mass Stop/Start Cloud Foundry Apps
+
+Table of Contents
+
+* [Description](#description)
+* [Requirements](#requirements)
+* [How to use](#how-to-use)
+
+## Description
+
+If your BTP global account uses the consumption-based commercial model, Cloud Foundry applications are billed based on their usage. Therefore, you can reduce the monthly cost by stopping applications which are used only for testing and development purposes. With the help of SAP Automation Pilot, such resources can be automatically stopped outside of working hours and started again the next day. To find out more about the Cloud Foundry's billng costs, please feel free to consult [SAP BTP, Cloud Foundry Runtime](https://discovery-center.cloud.sap/serviceCatalog/cloud-foundry-runtime?tab=service_plan&region=all&commercialModel=cpea).
+
+SAP Automation Pilot has the capabilities to perform mass operations on tens or even hundreds of Cloud Foundry applications, including stop, start or restart. In addition to this, these operations can be performed regularly with the [Scheduled Executions functionality](https://help.sap.com/docs/AUTOMATION_PILOT/de3900c419f5492a8802274c17e07049/96863a2380d24ba4bab0145bbd78e411.html).
+
+This example includes a total of four commands. *MassStopCfAppsInSpace* and *MassStartCfAppsInSpace* dynamically find and stop/start all applications in the given space. Similarly, *MassStopCfAppsInOrg* and *MassStartCfAppsInOrg* do the same thing but on organization level - the applications across all spaces are collected and then started/stopped. In addition, it's possible to ignore certain spaces and not perform any operations on them.
+
+![Mass Stop Apps in Org](./assets/mass-stop-org.png)
+![Mass Start Apps in Org](./assets/mass-start-org.png)
```

Comment:
what do you think about adding also the output of a succesful execution , see below?
![image](https://github.com/SAP-samples/automation-pilot-examples/assets/26586839/ffff91d8-4635-4f51-8dd6-7855213819e0)